src/net/sourceforge/kolmafia/session/EquipmentManager.java

```diff
@@ -924,6 +927,24 @@ public static final void addConditionalSkills(final Slot slot, AdventureResult i
         KoLCharacter.addAvailableSkill(SkillPool.SWOOP_LIKE_A_BAT);
         KoLCharacter.addAvailableSkill(SkillPool.SUMMON_CAULDRON_OF_BATS);
       }
+      case ItemPool.SHERIFF_BADGE -> {
```

Comment:
I would combine all these cases and add a check for all three, to make it a little more DRY

Reply:
How? I was asked to use Bear Arms as a model and they're separated.

Reply:
you could check for all three being equipped whenever any one is equipped

I didn't do it that way because I wasn't sure whether Mafia set the item you're trying to equipped as "equipped" before it finished running the conditional skill branch, and because splitting them is slightly more performant (as you know that the item you're currently equipping is equipped, so you save an operation)

Reply:
Yeah I figured the performance here wasn't too important vs maintainability. I think I'm right but am happy to be overruled

Reply:
conversely: each case statement is so simple to understand that the maintainability gain is insignificant, and the optimisation is so simple it may as well be done

also for this in particular, we're ~never going to edit this function so the maintainability gain is zero, whereas any perf gain will affect everybody who runs the function, so it's worthwhile even though it's trivial (i.e. basically also zero) and dwarfed by mafia's lack of performance in other areas.

Reply:
That depends on your definition of maintainability. This runs once upon equipping an item, it has no performance implication at all. But as I said, happy to let it go through. It is definitely stylistic.